src/main/java/org/opensearch/security/dlic/rest/api/WhitelistApiAction.java

```diff
@@ -36,23 +36,23 @@
  * SuperAdmin certificate for the default superuser is stored as a kirk.pem file in config folder of OpenSearch
  * <p>
  * Example calling the PUT API as SuperAdmin using curl (if http basic auth is on):
- * curl -v --cacert path_to_config/root-ca.pem --cert path_to_config/kirk.pem --key path_to_config/kirk-key.pem -XPUT https://localhost:9200/_opendistro/_security/api/whitelist -H "Content-Type: application/json" -d’
+ * curl -v --cacert path_to_config/root-ca.pem --cert path_to_config/kirk.pem --key path_to_config/kirk-key.pem -XPUT https://localhost:9200/_security/api/whitelist -H "Content-Type: application/json" -d’
```

Comment:
There's other instances like here too.

Reply:
Got it. Do we need to introduce another API like the [migrate API](https://github.com/opensearch-project/security/blob/2.x/src/main/java/org/opensearch/security/dlic/rest/api/MigrateApiAction.java) to handle cases where a cluster may still be referencing `whitelist.yml`?